src/models/commentModel.js

```diff
@@ -1,14 +1,29 @@
 import mongoose from 'mongoose';
+const { Schema } = mongoose;
 
-const CommentSchema = new mongoose.Schema({
-  id: {
+const commentSchema = new mongoose.Schema({
+  content: {
     type: String,
+    required: true
+  },
+  author: {
+    id: {
+      type: Schema.Types.ObjectId,
+      ref: 'User',
+      required: true
+    },
+    commenterName: String
```

Comment:
still not sure if this is necessary. u shouble be able to get this by the authorId.

Reply:
I have already moved the "commenterName" inside the author. I would like to keep it for now, which is not a big deal.